CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Enhancements
 ### Bug Fixes
 ### Infrastructure
+- Add backward test cases for neural sparse two phase processor.
```

Comment:
please make a proper changelog entry - pr number and link are missing

Reply:
Okay, have added it to the change log.